src/WebJobs.Extensions.EasyTables/EasyTableAttribute.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace WebJobs.Extensions.EasyTables
```

Comment:
Good point. Done.

---
In reply to: [54792936](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/45#discussion_r54792936) [](ancestors = 54792936)